src/App.js

```diff
@@ -16,9 +16,7 @@ export default function App() {
     // Visitor ID is the unique string ID used to identify a visitor of the site.
     const [visitorId, setVisitorId] = useState("");
     // vistorHistory is the returned data from the SERVER API call.
-    const [visitorHistory, setVisitorHistory] = useState("");
-    // This is just to display the response title (underneath the button)
-    const [responseSummary, setResponseSummary] = useState("");
+    const [visitorHistory, setVisitorHistory] = useState({});
```

Comment:
not sure if initializing it with an empty object is useful, see the last comment in this review